content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,208 @@
+=============
+Configuration
+=============
+
+The following document goes through all of the configurations and settings for the *Fleet*
+application, including settings, manufacturers, models, and categories.
+
+Settings
+========
+
+To access the settings menu, go to :menuselection:`Fleet application --> Configuration -->
+Settings`. There are only two settings that need configuration, the vehicle contract alert, and
+a new vehicle request.
+
+End Date Contract Alert
+-----------------------
+
+To help ensure no contracts lapse or vehicles are left without contracted drivers, Odoo has an
+alert in place to let the fleet manager and driver know when a contract is abut to expire. Enter
+the number of days before a contract is going to expire that an alert email will be sent.
+
+New Vehicle Request
+-------------------
+
+Set a limit on how many vehicle are in the fleet by setting a limit on requests. Employees will not
+be able to request a new car if there is a specific amount of cars already in the fleet. Enter the
+number limit for existing cars in this field.
+
+.. image:: configuration/fleet-settings.png
+   :align: center
+   :alt: Settings available for the Fleet application.
+
+Manufacturers
+=============
+
+Odoo Fleet comes pre-configured with sixty-six commonly used car manufacturers in the database,
```

Comment:
If you look in Runbot, one of the manufacturers is for a bicycle, not a car. Then, if you make a DB using 15.0 form scratch, it is just the 66 manufacturers. So the test data in runbot is adding a bicycle which is not the default. This took me a while to figure out in the beginning since I had added in that bike manufacturer at first, but it's misleading in Runbot.